web/template/layout/navigation.gotmpl

```diff
@@ -11,7 +11,7 @@
               <a class="moj-primary-navigation__link" {{ if eq .Path "/teams" }}aria-current="page"{{ end }} href="{{ prefix "/teams" }}">Teams</a>
             </li>
             <li class="moj-primary-navigation__item">
-              <a class="moj-primary-navigation__link" {{ if eq .Path "/my-details" }}aria-current="page"{{ end }} href="{{ prefix "/my-details" }}">My details</a>
+              <a class="moj-primary-navigation__link" {{ if eq .Path "/my-details" }}aria-current="page"{{ end }} href="{{ prefix "/my-details" }}">Change phone number</a>
```

Comment:
I don't think this should change: the page it links to provides all of a users details for reference, not just the phone number changing form